content/cover_letter.md

```diff
@@ -0,0 +1,26 @@
+Scott Markel
+Dassault Systèmes BIOVIA
+5005 Wateridge Vista Dr
+San Diego, CA 92121
+
+October Xth, 2020
+
+Dear Dr. Markel,
+
+We are writing to submit our manuscript entitled “Ten Quick Tip for Deep Learning in Biology” for consideration in _PLOS Computational Biology_'s education section.
```

Comment:
This is a good letter draft. I would suggest to add some motivation for writing/submitting this article. Here is a quick write up of what I have in mind. I suggest sandwiching this between the two paragraphs (after "We are writing ..." and before "Ultimately, we have ...":

> Deep learning provides new opportunities for classifying and recognizing patterns in biological data. While deep learning is a relatively young field within the broader field of machine learning, many general-purpose algorithms and methods shared by the broader deep learning community can be readily repurposed and applied to biological data. While we encourage the use of deep learning to solve existing and novel problems in biology, we want to share essential tips among practitioners for avoiding common pitfalls.

Reply:
Small edits: "Ten Quick Tip" -> "Ten Quick Tips"

Reply:
Shoot. Missed that. Going to push directly to master

Reply:
May also want to emphasize any prior communication with journal, collaborative nature of this manuscript etc at the beginning,